signer/local/local.go

```diff
@@ -133,26 +133,18 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 			cferr.BadRequest, errors.New("not a certificate or csr"))
 	}
 
-	template, err := signer.ParseCertificateRequest(s, block.Bytes, req.Subject)
+	template, err := signer.ParseCertificateRequest(s, block.Bytes, req.Subject, req.Hosts)
```

Comment:
ParseCertificateRequest already did the hosts appending, remove the duplicate code below.